src/Microsoft.Data.Analysis/Microsoft.Data.Analysis.csproj

```diff
@@ -10,6 +10,9 @@
     <Description>This package contains easy-to-use and high-performance libraries for data analysis and transformation.</Description>
     <PackageReleaseNotes>Initial preview of robust and extensible types and algorithms for manipulating structured data that supports aggregations, statistical funtions, sorting, grouping, joins, merges, handling missing values and more. </PackageReleaseNotes>
     <PackageTags>ML.NET ML Machine Learning Data Science DataFrame Preparation DataView Analytics Exploration</PackageTags>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <!-- Documentation warnings -->
+    <NoWarn>$(NoWarn);1591</NoWarn>
```

Comment:
I haven't written documentation for public methods that override base class methods because the base class methods have documentation on them. Suppressing 1591 is not ideal, this disables warnings on all public methods, but I verified that only the overriden methods produce this error. I could open an issue to get rid of this suppresion later. For now, this will let us ship a 0.2.0 with all the documentation we already have.

Reply:
Could you use `#pragma disable warning` only around the affected areas rather than applying it globally to the whole project?

Also, how much effort is it to document the base classes too?

@carlossanlop - any suggestions for the docs warning with base classes? Do we know if there's a solution where the default comments will get used if one isn't provided?

Reply:
If the base class already has documentation, but you are going to override a method, then you should copy the comments for the base method and make the necessary changes to adapt it to this class that overrides it.

For example, Stream has basic documentation of what we expect a method to do, and the classes that inherit from Stream and override the base methods also have documentation, very similar but with small adaptations to describe the functionality of the method specifically for that class.

> _Do we know if there's a solution where the default comments will get used if one isn't provided?_

You could also use `inheritdoc`, which may get rid of the warning, although I am not entirely sure that it works (meaning, I don't think the comments get inherited when we build and generate the documentation xmls, it may be a bug on our side).

Reply:
I'll look into `inheritdoc` tomorrow. I made sure that all the base class methods have documentation on them. There are some template generated methods that I can cheaply get documentation on. With these 2 steps, I'm guessing most of the errors will go away.

Reply:
`inheritdoc` is your friend here.

> Suppressing 1591 is not ideal, this disables warnings on all public methods

We shouldn't suppress 1591 globally because any new undocumented public member won't cause an error.